contracts/exchange-forwarder/contracts/src/MixinAssets.sol

```diff
@@ -89,10 +116,9 @@ contract MixinAssets is
             msg.sender,
             amount
         ));
-        require(
-            success,
-            "TRANSFER_FAILED"
-        );
+        if (!success) {
+            LibRichErrors.rrevert(LibForwarderRichErrors.TransferFailedError());
```

Comment:
We can get the `returnData` from the above call (the second return value). It might be useful to pass that into `TransferFailedError`.